exercises/robot-name/src/test/scala/RobotNameTest.scala

```diff
@@ -1,31 +1,45 @@
 import org.scalatest._
 
+import collection.mutable
+
 class RobotNameSpecs extends FunSpec with Matchers {
   val nameRegex = """[A-Z]{2}\d{3}"""
 
   it ("has a name") {
-    new Robot().name should fullyMatch regex (nameRegex)
+    new Robot().name should fullyMatch regex nameRegex
   }
 
   it ("does not change its name") {
-    pending
     val robot = new Robot
     val name = robot.name
     robot.name should be (name)
   }
 
   it ("does not have the same name as other robots") {
-    pending
```

Comment:
Please keep the `pending`s.
This is how all test suites are designed: All but the first test are set to `pending`.